cfgov/jinja2/v1/_includes/molecules/info-unit.html

```diff
@@ -71,13 +71,20 @@
             <ul class="list list__links">
             {% for link in value.links %}
                 <li class="list_item">
+                    {% if link.text is defined and link.text.find('@') > -1 %}
```

Comment:
[Here's](http://emailregex.com/) a somewhat hardcore regex for matching email addresses.